scss/detail.scss

```diff
@@ -56,29 +72,28 @@ $max-width-mobile: 1024px;
 } 
 
 .container-item {
-  margin: 25px;
+  margin: 15px 30px;
 }
 
 .container-data {
-  flex-grow: 2;
+  grid-area: description;
+  height: 65%;
```

Comment:
I am no sure for this value, is relative ??

Reply:
yes, is relative